rotkehlchen/tasks/manager.py

```diff
@@ -807,6 +810,37 @@ def _maybe_query_monerium(self) -> Optional[list[gevent.Greenlet]]:
             method=monerium.get_and_process_orders,
         )]
 
+    def _maybe_trigger_calendar_reminder(self) -> Optional[list[gevent.Greenlet]]:
+        """Get upcoming reminders and maybe process them"""
+        with self.database.conn.read_ctx() as cursor:
```

Comment:
Two things about this function.

1. Shouldn't it run every X minutes? This will now get always scheduled and at least the sqlite query will run continuously. I know this should indeed run frequently but wondering if perhaps a variable in the task manager (in-memory variable) to remember last query would be fine, to say do this check every 5 mins, and not every time this is scheduled. Not sure here ...
2. How/when do events that have passed get deleted?

Reply:
The events get deleted after the ws is sent in the function that gets called?

Reply:
I don't see that.